schemas/common/definitions.json

```diff
@@ -63,6 +63,11 @@
     "description": "The channel used to launch the EQ.",
     "examples": ["RAS", "RH"]
   },
+  "source": {
+    "$ref": "string",
```

Comment:
The key needs to be `type` instead of `$ref`. I also think we should use `enum` here: So `"enum": ["SRM"]`. It can be expanded in future. Currently, this allows any arbitary string.